src/quacc/schemas/ase.py

```diff
@@ -516,3 +600,39 @@ def vib_and_thermo(
             gzip_file=self._settings.GZIP_FILES,
             store=store,
         )
+
+
+def _get_nth_iteration(
+    neb_trajectory: list[Atoms], n_iter: int, n_images: int, interval: int
+) -> list[Atoms]:
+    """
+    Extract every nth iteration from the NEB trajectory.
+
+    Parameters
+    ----------
+    neb_trajectory
+        List of configurations (length: n_iter * n_images).
+    n_iter
+        Total number of iterations.
+    n_images
+        Number of images per iteration.
+    interval
+        Interval to get every nth iteration.
+
+    Returns
+    -------
+    list[Atoms]
+        List of configurations from every nth iteration.
+    """
+    result = []
+    if interval > n_iter:
+        pass
```

Comment:
@kumaranu: FYI that this code block doesn't actually do anything. I'm going to remove it, but if you meant for it to do something else just let me know.